<launch>
    <group ns="robot1">
        <node pkg="image_processing" type="detect_aruco.py" name="detect_aruco">
            <remap from="detect_image" to="/robot1/detect_image"/>
            <remap from="ar_point" to="/robot1/ar_point"/>
            <remap from="/zed_node/left/image_rect_gray" to="/robot1/zed_nodelet/left/image_rect_gray"/>
        </node>
        <node pkg="image_processing" type="coordinate_marker_camera1" name="coordinate_marker_camera1"/>
    </group>
</launch>
